src/components/Nav/RegistrationForm.jsx

```diff
@@ -97,80 +95,78 @@ export default function RegistrationForm() {
-                ) : null}
-                <FormText>
-                  Needs at least 6 characters &amp; is required.
-                </FormText>
-              </FormGroup>
-              <FormGroup>
-                <Label htmlFor="Email">Email</Label>
+              ) : null}
+              <FormText>
+                Needs at least 6 characters &amp; is required.
+              </FormText>
+            </FormGroup>
+            <FormGroup>
+              <Label htmlFor="Email">Email</Label>
+              <Input
+                type="primaryemail"
+                name="primaryemail"
+                id="Email"
+                value={formState.primaryemail}
+                onChange={inputChange}
+                placeholder="your@primaryemail.com"
+              />
+              {errors.primaryemail.length > 0 ? (
+                <AlertRed
+                  message={<p className="error">{errors.primaryemail}</p>}
+                />
+              ) : null}
+              <FormText>Required. We will never send spam.</FormText>
+            </FormGroup>
+            <FormGroup check>
+              <Label check>
                 <Input
-                  type="primaryemail"
-                  name="primaryemail"
-                  id="Email"
-                  value={formState.primaryemail}
+                  type="checkbox"
+                  name="terms"
+                  value={formState.terms}
                   onChange={inputChange}
-                  placeholder="your@primaryemail.com"
-                />
-                {errors.primaryemail.length > 0 ? (
-                  <AlertRed message={<p className="error">{errors.primaryemail}</p>} />
+                />{" "}
+                I Agree to Terms of Service and promise to be a nice human.
+                {errors.terms.length > 0 ? (
+                  <AlertRed message={<p className="error">{errors.terms}</p>} />
                 ) : null}
-                <FormText>Required. We will never send spam.</FormText>
-              </FormGroup>
-              <FormGroup check>
-                <Label check>
-                  <Input
-                    type="checkbox"
-                    name="terms"
-                    value={formState.terms}
-                    onChange={inputChange}
-                  />{" "}
-                  I Agree to Terms of Service and promise to be a nice human.
-                  {errors.terms.length > 0 ? (
-                    <AlertRed
-                      message={<p className="error">{errors.terms}</p>}
-                    />
-                  ) : null}
-                </Label>
-                <FormText>
-                  After logging in, you may edit more profile options.
-                </FormText>
-              </FormGroup>
-            </Form>
+              </Label>
+              <FormText>
+                After logging in, you may edit more profile options.
+              </FormText>
+            </FormGroup>
```

Comment:
I think all that changed here is the removal of the seemingly redundant <Form> <Form/> again. git is showing it weirdly epic though, so not sure if i'm missing anything.